lib/omniauth/strategies/namecheap.rb

```diff
@@ -10,6 +10,10 @@ class Namecheap < OmniAuth::Strategies::OAuth2
                               token_url:     '/apps/sso/api/token',
                               me_url:        '/apps/sso/api/resource/user'
 
+      def callback_url
+        full_host + script_name + callback_path
```

Comment:
Looking at the source for [this method](https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategy.rb#L439), it seems that we're removing the query string here. Could you add a test that shows it?